### PR TITLE
Support payload for URL actions

### PR DESCRIPTION
**Pull request by digitkhrisnaa**
I hereby agree to the terms of the CLA available at: [link](https://yandex.ru/legal/cla/?lang=en)

### Notes:
- Add payload into `DivActionInfo`
- Update reporter `DivActionHandler` to pass the action payload.

### CODE REVIEW

**Comment by lunarstill** [comment]
How do you want use this logger? It remains from legacy clients. Consider using `DivReporter` for actions logging.

File: client/ios/DivKit/Actions/DivActionHandler.swift [hunk 1]
```diff
@@ -187,11 +187,13 @@ public final class DivActionHandler {
     )
 
     if !isDivActionURLHandled {
+      let referer = action.resolveReferer(expressionResolver)
+      logger.log(url: url, referer: referer, payload: action.payload)
```
Comment (lunarstill) [inline client/ios/DivKit/Actions/DivActionHandler.swift:191]: Seems like a breaking change. All non div-action urls will be sent to DefaultDivActionLogger (which actually makes http-requests). I don't think it is a good idea.

**Comment by digitkhrisnaa** [comment]
> How do you want use this logger? It remains from legacy clients. Consider using `DivReporter` for actions logging.

Hi, DivKit has a payload in both actions and visibility_actions, but there is no protocol for that in the library. `DivReporter` only sends the source and log_id, while `DivUrlHandler` delegates the URL only. Therefore, the proposal is to log any custom actions with the payload. Or is there any workaround for that which I'm missing in DivKit?

**Comment by lunarstill** [comment]
I think it is better to add payload into `DivActionInfo` that is used in `DivReporter`.

**Comment by digitkhrisnaa** [comment]
Updated by adding payload into `DivActionInfo` and reverted the previous changes

**Comment by lunarstill** [comment]
Merged: https://github.com/divkit/divkit/commit/1c76a80aa8503b2a2a2c282ebba51428930862a1

**Pull request closed without merging**